forui/lib/src/widgets/select_group/select_group_controller.dart

```diff
@@ -0,0 +1,94 @@
+import 'package:flutter/widgets.dart';
+
+/// A controller for a select group.
+abstract class FSelectGroupController<T> with ChangeNotifier {
```

Comment:
Should we make all fields & methods in this class abstract (make it effectively an interface), and remove the `values` getter?

The radio & multiple select controllers implementations seem to be quite different. Doing so will simplify the radio controller by allowing it to use a single nullable value.

Reply:
I don't think it makes much a difference. We're just pushing the complexity to other places. For example, we'll need to reimplement more methods, `contains()` and `values()`.

I believe we could still use this as an interface in dart if there's some usecase where you'd like to reimplement everything. So multiple inheritance shouldn't be an issue here.